ui/pages/token-details/index.scss

```diff
@@ -0,0 +1,50 @@
+.token-details {
+  width: 359px;
+
+  @media screen and (max-width: $break-small) {
+    .page-container {
+      max-height: 82vh;
+      margin-right: auto;
+      margin-left: auto;
+    }
+  }
+
+  &__title {
+    text-transform: capitalize;
+  }
+
+  &__closeButton {
+    background: none;
+    float: right;
+    margin-top: -8px;
+    margin-right: -4px;
```

Comment:
`margin-inline-end` would be better for RTL support.

Reply:
@darkwing Fixed in: ad52edb29d98d72599f0cb307501e19525d57d36